Tests/API/NodeTests.cs

```diff
@@ -0,0 +1,161 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using System.Web;
+using NUnit.Framework;
+using QuantConnect.Api;
+using QuantConnect.API;
+using QuantConnect.Brokerages;
+using QuantConnect.Configuration;
+
+
+namespace QuantConnect.Tests.API
+{
+    [TestFixture, Ignore("These tests require an account, token, and organization ID")]
+    public class NodeTests
+    {
+        private int _testAccount = 1;
+        private string _testToken = "ec87b337ac970da4cbea648f24f1c851";
+        private string _testOrganization = "enter Org ID here";
+        private string _dataFolder = Config.Get("data-folder");
+        private Api.Api _api;
+
+        /// <summary>
+        /// Run before every test
+        /// </summary>
+        [SetUp]
+        public void Setup()
+        {
+            _api = new Api.Api();
+            _api.Initialize(_testAccount, _testToken, _dataFolder);
+        }
+
+        /// <summary>
+        /// Test to creating a New Node
+        /// </summary>
+        [Test]
+        public void CreateNewNode()
+        {
+            string sku = Node.GetSKU(2, 8, "backtest");
+            var nodeName = $"{DateTime.UtcNow.Minute}:{DateTime.UtcNow.Second}-Pinocho";
+            var newNode = _api.CreateNode(nodeName, _testOrganization, sku);
```

Comment:
Create a type for SKU and enum for the types of nodes; cannot use string-encoded data as a parameter
and then use sku.ToString() to get the API transit version.

Reply:
New SKU class and NodeType enum in Nodes.cs. Now you must create a SKU object to call on CreateNode.